dev/com.ibm.ws.microprofile.health.2.0_fat/fat/src/com/ibm/ws/microprofile/health20/fat/DelayAppStartupHealthCheckTest.java

```diff
@@ -193,7 +193,19 @@ public void run() {
                             repeat = false;
                             startServerThread.join();
                         } else if (System.currentTimeMillis() - start_time > time_out) {
-                            throw new TimeoutException("Timed out waiting for server and app to be ready. Timeout set to " + time_out + "ms.");
+
```

Comment:
In the if statement in lines 191-194, it would be good to add some log messages, so it would be easier to see if the readiness endpoint response code was 200.